pkgs/development/haskell-modules/configuration-ghc-9.8.x.nix

```diff
@@ -131,8 +131,10 @@ self: super: {
   hip = appendConfigureFlag "--ghc-options=-fsimpl-tick-factor=200" super.hip;
 
   # Fix build with text-2.x.
-  libmpd = appendPatch (pkgs.fetchpatch
-      { url = "https://github.com/vimus/libmpd-haskell/pull/138.patch";
+  libmpd = appendPatch
```

Comment:
attribute path of this is `haskell.packages.ghc98.libmpd` for the commit message.